code/modules/mob/living/emote.dm

```diff
@@ -141,6 +141,10 @@
 	key_third_person = "flaps"
 	message = "flaps their wings."
 	hands_use_check = TRUE
+	mob_alt_emotes = list(
+		/mob/living/carbon/human/species/moth = "flaps their wings.",
```

Comment:
This will not work, try this on a *real* player moth, not one you spawned in.

Every player character is a `/mob/living/carbon/human`. These only exist for easy generation.

Change this to `/datum/species`, and match on `dna.species`.